Packs/CortexXDR/ReleaseNotes/4_1_6.md

```diff
@@ -0,0 +1,4 @@
+
+#### Playbooks
+##### Cortex XDR incident handling v3
+- New resolve status when closing remediated Cortex XDR incident - RESOLVED_TRUE_POSITIVE. 
```

Comment:
Please link to their release notes/explain why we changed the status. 

Also I think it should be resolved status, not resolve status.